packages/neuron-ui/src/components/History/index.tsx

```diff
@@ -1,3 +1,86 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
+import { DataTable, Text } from 'grommet'
+import { Transaction } from 'grommet-icons'
+import styled from 'styled-components'
+import TablePagination from './tablePagination'
 
-export default () => <h1>History</h1>
+interface Transaction {
+  date: string
+  value: number
+  hash: string
+}
+
+const FooterDiv = styled.div`
+  height: 80px;
+`
+
+const formatterDate = (date: Date) => {
+  const y = date.getFullYear()
+  const m = date.getMonth()
+  const d = date.getDate()
+  return `${y}-${m < 10 ? `0${m}` : m}-${d < 10 ? `0${d}` : d} ${date.toTimeString().substr(0, 8)}`
+}
+
+const requestHistory = (_typeHash: string, page: number, pageSize: number) => {
+  const list = []
+  for (let i = 0; i < pageSize; i += 1) {
+    const num = pageSize * page + i
+    const date = new Date(new Date().getTime() - num * 1000 * 3600)
+    list.push({
+      date: formatterDate(date),
+      value: 300.34 + num,
+      hash: `0x37cc18636c632fd8ad0d0682b33a1dff8c2b621be29e82403c898cecabb2ec9e${num.toString()}`,
+    })
+  }
+  return list
+}
+
+const History = () => {
+  const walletAddress = '0x22f5Bd068FE2F7bEd72D8c9c8CC22e872AD48Bb9'
+  const pageSize = 14
+  const [page, setPage] = useState(0)
+  const [list, setList] = useState(requestHistory(walletAddress, page, pageSize))
+  const total = 200
+
+  useEffect(() => {
+    setList(requestHistory(walletAddress, page, pageSize))
+  }, [page])
```

Comment:
It's ok but not good, once the request failed, the list will not be updated while the page is updated.

But leave it now, the effect will be moved to top container.